### ISSUE TITLE
[Feature Request] Import ggplot code

### ISSUE THREAD

**Issue by sanojkl** [dreamRs/esquisse#235]
When not working in one session, you can export the code but couldn't reimport it

**Comment by XuejingJiang** [comment]
Yes, I would also like to see this feature become available if possible.

This would be really helpful to edit an existing plot when we already have the ggplot codes and do not need to recreate the plot by going through all the point and click steps. If achievable, I think that this will be the best mix of reproducible coding and simple UI for some users!
